src/components/vm/consoles/vnc.jsx

```diff
@@ -400,8 +433,8 @@ export class VncActive extends React.Component {
                           onInitFailed={this.onInitFailed}
                           onSecurityFailure={this.onSecurityFailure}
                           consoleContainerId={isExpanded ? "vnc-display-container-expanded" : "vnc-display-container-minimized"}
-                          resizeSession
-                          scaleViewport
```

Comment:
now that I've seen the video, this old code really doesn't make sense at all :sweat:

Reply:
Not really... :-)